packages/core/src/sync/index.ts

```diff
@@ -350,12 +350,13 @@ export const createSync = async (params: {
           for await (const { events, checkpoint } of eventGenerator) {
             // Sort out any events before the crash recovery checkpoint
             if (
+              params.crashRecoveryCheckpoint &&
```

Comment:
Before, we used the zero checkpoint string as the empty value here, correct? So, this diff should have no functional change?

Reply:
Correct. Now we use `undefined` to represent an app with no crash recovery instead of `zeroCheckpoint`.